tide_core.install

```diff
@@ -639,7 +639,9 @@ function tide_core_update_8021() {
   $config_path = drupal_get_path('module', 'tide_core') . '/config/install';
   $source = new FileStorage($config_path);
   $config_storage = \Drupal::service('config.storage');
-  $config_storage->write($view, $source->read($view));
+  if ($source->read($view)) {
```

Comment:
as the hook already in there and the view is removed from install directory, this will avoid throwing error during tide_core build in CI